vm_automation/esxiVm.py

```diff
@@ -324,13 +317,13 @@ def getFileFromGuest(self, srcFile, dstFile):
                                           " AS " + dstFile + \
                                           " HTTP RESPONSE WAS " + str(resp.status_code))
                         retVal=True
-                except Exception as e:
-                    self.server.logMsg(str(e))
+                except vim.fault.FileNotFound as e:
+                    self.server.logMsg("FAILED TO FIND FILE ON VM: " + srcFile)
+                    self.server.logMsg("SYSTEM ERROR: " + str(e))
                     pass
-                #SOMETIMES THE VM GETS SQUIRRLEY; IF SO, JUST TRY AGAIN
                 except Exception as e:
                     self.server.logMsg("UNPREDICTED EXCEPTION:\n" + str(e))
-                    continue
+                    pass

```

Comment:
There is no code to execute after the `pass` (same for line 323) should this just be removed? Or does pass somehow nullify the exception handling is some way not documented clearly?

https://www.tutorialspoint.com/python/python_pass_statement.htm